open_lm/model.py

```diff
@@ -254,6 +278,8 @@ def create_model(args):
         weight_tying=cfg["weight_tying"],
         norm_type=get_norm_class(args),
         apply_qk_norm=args.qk_norm,
-        rotary_old=args.rotary_old
+        mup_base_dim=args.mup_base_dim,
+        mup_base_n_heads=args.mup_base_n_heads,
+        #rotary_old=args.rotary_old
```

Comment:
Why is rotary old removed?

Reply:
also is there any way we can make minimal (ideally no) changes to the model.py file? mup is kind of a niche use case for open lm, so it might be nice to have the implementation separate from the main model. maybe we can use some mixin pattern for mup instead?